### DISCUSSION TITLE
desktop.ini question

### DISCUSSION THREAD

**Discussion by GT500org** [upscayl/upscayl#650] [Q&A]
In the 2.9.8 release notes, when you say "Upscayl will now delete desktop.ini", I assume you mean it deletes it from the batch upscale queue and not that it deletes the desktop.ini file from the filesystem, right?

**Comment by NayamAmarshe** [comment]
Well, it's kinda hacky. It's gonna delete the desktop.ini file only in the batch upscayl folder that you provide to it.

In the next update, this will be handled by the backend system, but for now, we're doing it from the frontend. We've also been doing the same for .DS_Store files on MacOS, so I just added Windows to the list now.

It's not deleting anything in the nested folders. Only the folder that contains the upscale files.

**Reply by GT500org** [reply]
I know that desktop.ini isn't technically an important system file, but it does contain settings for the folder it's in, so each time you delete a copy of it you reset all settings for that folder. An ideal workaround would be one that leaves the file intact and untouched.

**Reply by NayamAmarshe** [reply]
> I know that desktop.ini isn't technically an important system file, but it does contain settings for the folder it's in, so each time you delete a copy of it you reset all settings for that folder. An ideal workaround would be one that leaves the file intact and untouched.

Yes of course, you're not wrong but we're assuming you're creating a special folder for batch upscayl.
For the time being, this is a good enough solution for our needs but of course, the backend needs to be updated for a permanent and better solution.

**Reply by aaronliu0130** [reply]
There was a problem where having a .ini file would stop the entire batch upscayl.
